crates/turborepo-ui/src/tui/app.rs

```diff
@@ -126,6 +131,18 @@ impl<W> App<W> {
         })
     }
 
+    fn update_sidebar_toggle(&mut self) {
+        let value = !self.preferences.is_task_list_visible();
+        self.preferences.set_is_task_list_visible(Some(value));
+    }
+
+    fn update_task_selection_pinned_state(&mut self) -> Result<(), Error> {
+        // Preferences assume a pinned state when there is an active task.
+        // This `None` creates "un-pinned-ness" on the next TUI startup.
+        self.preferences.set_active_task(None)?;
+        Ok(())
+    }
```

Comment:
I find this name a little confusing. Does `clear_pinned_task` express the behavior better?
```suggestion
    fn update_task_selection_pinned_state(&mut self) {
        // Preferences assume a pinned state when there is an active task.
        // This `None` creates "un-pinned-ness" on the next TUI startup.
        self.preferences.set_active_task(None);
    }
```